.github/workflows/sdk-regression.yml

```
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
```

```suggestion
    types: [created, edited]
```

* lets allow edited comment as well
  *  say we edited the comment later to correct name post some typo -> it should trigger this workflow.